react/fields/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.label {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    font-weight: @sk-bold;
+  }
+}
+
+.secondaryLabel {
+  font-weight: @sk-light;
```

Comment:
Let's stick with the default way it currently is now, I don't want to be kind of half tweaking this change on individual components. It really should be formed as patterns on a higher level.

Thanks!